codesniffer.ruleset.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<ruleset name="WordPress Theme Coding Standards">
+	<!-- See https://github.com/squizlabs/PHP_CodeSniffer/wiki/Annotated-ruleset.xml -->
+	<!-- See https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/develop/WordPress-Core/ruleset.xml -->
+
+	<!-- Set a description for this ruleset. -->
+	<description>A custom set of code standard rules to check for WordPress themes.</description>
+
+	<!-- Include the WordPress ruleset, with exclusions. -->
+	<rule ref="WordPress">
```

Comment:
@philiparthurmoore this should be `WordPress-Core` instead, as `WordPress` is a superset of all the standards and it has a lot of extras that wouldn't be applicable.

There's also, actually, a [`WordPress-Docs` standard](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/develop/WordPress-Docs/ruleset.xml) now too which can be used to check for PhpDoc comments.

So I think this should be replaced with:

```
<rule ref="WordPress-Core" />
<rule ref="WordPress-Docs" />
```